Apps/Common/test/unit/CacheProviders/DistributedCacheProviderTests.cs

```diff
@@ -237,5 +269,28 @@ private static string GenerateRandomString()
         {
             return Guid.NewGuid().ToString()[..5];
         }
+
+        private static Task<object?> GetGeneratedItemValue(CacheType cacheType)
```

Comment:
This doesn't need to return a Task.  One of the tests that call this method puts the returned value inside a task: `Task.FromResult(value)` (line 243) while the other does not (line 146).  It would be simpler if this method returned just `object?` and the `Task.FromResult` was in both calling methods.

I'd also suggest updating the new tests' names to indicate they are testing the default value behaviour for cache misses.